src/DucksNet.WebUI/wwwroot/css/sign_up_design.css

```diff
@@ -0,0 +1,65 @@
+body {
+    border: none;
+    height: 75vh;
+    width:75vw;
+}
```

Comment:
Nu cred ca e ok sa facem selectors pe body. Ca pagina asta o sa fie practic I injectata in main